### PR TITLE
Jac/publish samples

### PR DESCRIPTION
**Pull request by jacalata**
Added an optional parameter allowing users to publish sample content into a project on creation or update, as described in the REST API docs https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_projects.htm#create_project

### CODE REVIEW

File: samples/create_project.py [hunk 1]
```diff
@@ -45,23 +45,28 @@ def main():
...
+    server = TSC.Server(args.server)
+    server.add_http_options({'verify': False})
+    server.use_server_version()
     with server.auth.sign_in(tableau_auth):
         # Use highest Server REST API version available
         server.use_server_version()
 
         # Without parent_id specified, projects are created at the top level.
-        top_level_project = TSC.ProjectItem(name='Top Level Project')
+        top_level_project = TSC.ProjectItem(name='TPL2')
```
Comment (vogelsgesang) [inline samples/create_project.py:56]: why rename? I actually preferred the previous names...
Here and other places

File: samples/create_project.py [hunk 2]
```diff
@@ -45,23 +45,28 @@ def main():
     logging.basicConfig(level=logging_level)
 
     tableau_auth = TSC.PersonalAccessTokenAuth(args.token_name, args.token_value, site_id=args.site)
-    server = TSC.Server(args.server, use_server_version=True)
+    server = TSC.Server(args.server)
+    server.add_http_options({'verify': False})
+    server.use_server_version()
```
Comment (vogelsgesang) [inline samples/create_project.py:50]: why disable verification in the sample?
I don't think we should such an anti-pattern/workaround into a sample which many customers might use as a starting point for their own scripts

File: samples/create_project.py [hunk 3]
```diff
@@ -45,7 +45,8 @@ def main():
     logging.basicConfig(level=logging_level)
 
     tableau_auth = TSC.PersonalAccessTokenAuth(args.token_name, args.token_value, site_id=args.site)
-    server = TSC.Server(args.server, use_server_version=True)
+    server = TSC.Server(args.server)
+    server.use_server_version()
```
Comment (vogelsgesang) [inline samples/create_project.py:49]: why change this?

**Review by bcantoni — CHANGES_REQUESTED** [review]
Ah good point @vogelsgesang. This came up in #859 and we wrote a standalone issue to discuss whether we should change the default (see #876). I think we should revert this bit as part of this PR and revisit later.

**Review by bcantoni — APPROVED** [review]

File: samples/create_project.py (hunk 3 above)
Comment (jacalata) [reply samples/create_project.py:49]: imperfect undo. But it does make it easier to insert the ssl change for local testing :)

**Pull request merged**